splinepy/io/iges.py

```diff
@@ -35,4 +35,11 @@ def export(fname, splines):
     --------
     None
     """
+    # check if any spline contains volume elements
+    for ispl, spline in enumerate(splines):
+        if spline.para_dim == 3:
```

Comment:
```suggestion
        if spline.para_dim > 2 and spline.dim > 3:
```

Reply:
can you test two things with other CAD tools:
1. are spline.dim = 2 splines properly understood?
2. what happens with spline.dim = 1 splines?